src/button/src/components/styled/NeutralButtonStyles.js

```diff
@@ -0,0 +1,44 @@
+import { css } from 'styled-components';
+
+import { NEUTRALS, WHITE } from '../../../../colors';
+import { getStyleVariation } from '../../../../utils/StyleUtils';
+
+// border-top + border-bottom + padding-top + padding-bottom + line-height = 40px
+const fontColorVariation = getStyleVariation('fontColor', {
+  dark: NEUTRALS[0],
+  light: WHITE,
+});
+
+const neutralStyle = css`
+  background-color: transparent;
+  border-color: transparent;
+  color: ${fontColorVariation};
+  font-weight: 600;
+`;
+
+const neutralHover = css`
+  background-color: rgba(255, 255, 255, 0.2);
```

Comment:
We should probably do this with the other colors we have.

Reply:
As In use RGBA for all our colors and set opacity? I considered adding these to the `colors` file as consts but wasn't sure if that was really necessary.